src/ferc_xbrl_extractor/cli.py

```diff
@@ -99,11 +106,38 @@ def parse():
     return parser.parse_args()
 
 
+def load_extracted(
+    extracted: xbrl.ExtractOutput,
+    sqlite_uri: str,
+    duckdb_path: str | None,
+    clobber: bool,
+) -> None:
+    """Write extracted data to SQLite/Duckdb databases."""
+    engine = create_engine(sqlite_uri)
+    if clobber:
+        helpers.drop_tables(engine)
+
+    for table_name, data in extracted.table_data.items():
+        # Loop through tables and write to database
+        if not data.empty:
+            # Write to SQLite
+            with engine.begin() as sqlite_conn:
+                data.to_sql(table_name, sqlite_conn, if_exists="append")
+
+            # Write to duckdb if passed a path to duckdb
+            if duckdb_path is not None:
```

Comment:
nit: slightly annoying to have such deep nesting + conditionals here - it might be cleaner to have `write_to_sql` and `write_to_duckdb` as separate functions that get called in `main`. But really doesn't matter.